lib/puppet/provider/cs_location/crm.rb

```diff
@@ -18,7 +18,12 @@ def self.instances
     instances = []
 
     cmd = [ command(:crm), 'configure', 'show', 'xml' ]
-    raw, status = Puppet::Util::SUIDManager.run_and_capture(cmd)
+    if Puppet::PUPPETVERSION.to_f < 3.4
```

Comment:
I just copied what's already in all the other crm files. So for consistency this should be the same, or they should all be changed?

https://github.com/puppet-community/puppet-corosync/blob/master/lib/puppet/provider/cs_primitive/crm.rb#L90